src/crimson/os/seastore/seastore_types.h

```diff
@@ -215,6 +217,16 @@ constexpr paddr_t make_fake_paddr(segment_off_t off) {
 constexpr paddr_t zero_paddr() {
   return paddr_t{ZERO_SEG_ID, 0};
 }
+constexpr paddr_t delayed_temp_paddr(segment_off_t off) {
+  return paddr_t{DELAYED_TEMP_SEG_ID, off};
+}
+
+enum class delay_inline_ool_t {
```

Comment:
delay_inline_ool_t is almost a list of the possible values.  How about addr_allocation_type_t.